### DISCUSSION TITLE
Problem with two JK BMS

### DISCUSSION THREAD

**Discussion by rogergrant99** [mr-manuel/venus-os_dbus-serialbattery#80] [General]
I have two JK BMS using USB to RS-485, while setting the identifier on the first battery to 1 works with the bus driver Jkbms_pb , but the second battery (with identifier 2) will not show up, I modified the line in dbus-serialbattery.py  from :
{"bms": Jkbms_pb, "baud": 115200, "address": b"\x01"},
to 
{"bms": Jkbms_pb, "baud": 115200, "address": b"\x02"},
And it does not connect to the second battery.... Isn't the address tied to the battery identifier ?  
Also what do you use to monitor serial traffic for debugging ?

Thanks

**Comment by mr-manuel** [comment]
Check here https://github.com/Louisvdw/dbus-serialbattery/discussions/969

**Comment by rogergrant99** [comment]
Thanks, switched to v1.4.20240902dev and added MODBUS_ADDRESSES = 0x01, 0x02 to config.ini and I see two batteries now.

**Comment by rogergrant99** [comment]
Thanks, switched to v1.4.20240902dev and added MODBUS_ADDRESSES = 0x01, 0x02 to config.ini and I see two batteries now.
